test/e2e/e2e_test.go

```diff
@@ -133,6 +146,67 @@ var _ = Describe("controller", Ordered, func() {
 				return nil
 			}
 			EventuallyWithOffset(1, verifyOneRunningPod, time.Minute, time.Second).Should(Succeed())
+
+			By("validating that the nginx pod is running as expected")
+			//var nodeName string
+			verifyOneRunningNginxPod := func() error {
+				pods, err := clientset.CoreV1().Pods("ingress-nginx").List(ctx, v1.ListOptions{
+					LabelSelector: "app.kubernetes.io/component=controller", Limit: 1,
+				})
+				ExpectWithOffset(1, err).NotTo(HaveOccurred())
+				if len(pods.Items) != 1 {
+					return fmt.Errorf("got %d nginx pods", len(pods.Items))
+				}
+				if pods.Items[0].Status.Phase != "Running" {
+					return fmt.Errorf("nginx pod in %s status", pods.Items[0].Status.Phase)
+				}
+				fmt.Printf("nginx pod %s running on %s\n", pods.Items[0].Name, pods.Items[0].Spec.NodeName)
+				//nodeName = pods.Items[0].Spec.NodeName
+				return nil
+			}
+			EventuallyWithOffset(1, verifyOneRunningNginxPod, time.Minute, time.Second).Should(Succeed())
+			By("Verify PDB and PDBWatcher exist")
```

Comment:
leave a comment that this is different than the eviction-autoscaler manager in that the pdb is generated.